testing/cypress/e2e/externalLinks/english-external-links-validator.cy.js

```diff
@@ -0,0 +1,67 @@
+/// <reference types="Cypress" />
+
+
+const allPages = require("../../fixtures/testing-page.json");
+
+const excludedlinks = [
+  'https://voterregistration.ct.gov/OLVR/welcome.do?ref=voteusa_en',
+  // the above link will throw error code "read ECONNRESET" this will not pass through cypress test and has been checked manually but has not been added to `excluded-links.cy.js` test 
+  'https://voterservices.elections.maryland.gov/OnlineVoterRegistration/InstructionsStep1?ref=voteusa_en',
+  'https://www.sec.state.ma.us/ovr/?ref=voteusa_en',
+  'https://olvr.ohiosos.gov/?ref=voteusa_en',
+  'https://voterlookup.ohiosos.gov/voterlookup.aspx?ref=voteusa_en',
+  'https://olvr.hawaii.gov/?ref=voteusa_en',
+  'https://elections.hawaii.gov/voters/registration/?ref=voteusa_en',
+  'https://vote.sos.ri.gov/Home/RegistertoVote?ref=voteusa_en',
+  'https://vote.sos.ri.gov/Voter/RegisterToVote?ref=voteusa_en',
+  'https://vote.sos.ri.gov/Home/UpdateVoterRecord?ActiveFlag=0&?ref=voteusa_en',
+  // sc links
+  'https://vrems.scvotes.sc.gov/Voter/Login?ref=voteusa_en',
+  'https://scvotes.gov/voters/register-to-vote/?ref=voteusa_en',
+  'https://info.scvotes.sc.gov/eng/ovr/start.aspx?ref=voteusa_en'
+];
+
+describe("External Link Validator Test", () => {
+  const singlePage =
+    Cypress.env("name") && Cypress.env("route")
+      ? [
+          {
+            name: Cypress.env("name"),
+            route: Cypress.env("route"),
+          },
+        ]
+      : null;
+  const pages = singlePage !== null ? singlePage : allPages;
+  pages.forEach((page) => {
+    it( 
+      `${page.name === "" ? "home" : page.name}`,
+      () =>
+        Cypress.env("retries") === true
+          ? {
+              retries: {
+                runMode: 2,
+              },
+            }
+          : {},
+      () => {
+        cy.visit({
+          url: page.route,
+        });
+          // the follow code will be brought back in once content is on site
```

Comment:
right now this is just running to make sure everything is functioning... once we get content on the site il bring all this back